packages/bot-web-ui/src/components/dashboard/dashboard-components/cards.tsx

```diff
@@ -0,0 +1,64 @@
+//kept sometihings commented beacuse of mobx to integrate popup functionality here
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { Icon, Text } from '@deriv/components';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import LoadModalStore from 'Stores/load-modal-store';
+
+type TCard = {
+    load_modal: LoadModalStore;
+    active_tab: number;
+    setActiveTab: (active_tab: number) => void;
+};
+
+const Cards = ({ load_modal, setActiveTab }: TCard) => {
+    const { onDriveConnect } = load_modal;
+    const actions = [
```

Comment:
please consider memoizing this, since it will be created on prop changes and cause re-renders and since you are using `map index` as key, there is a high possibility that the whole `dom` will be repainted.

Reply:
done ✅